Controller/src/controller/regionalVM.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package controller
+
+import (
+	"github.com/golang/protobuf/proto"
+	"log"
+	"os/exec"
+)
+
+type regionalVM struct {
+	name string
+	zone string
+	cpuMin string
+	imageName string
+	active bool
+	probes []*ProbeConfig
+}
+
+func newRegionalVM(name string, zone string, cpu string, img string) *regionalVM {
+	ret := new(regionalVM)
```

Comment:
you can do all this in line like:

ret := &regionalVM {
 name: name,
zone: zone,
}

etc.

Reply:
addressed in 6a760db